lib/si-pkg/src/spec/func.rs

```diff
@@ -70,6 +70,7 @@ pub enum FuncSpecBackendKind {
     JsAttribute,
     JsAuthentication,
     Json,
+    // NOTE(nick): this has been deprecated. Not adding serde deprecated tag in case it affects the type.
```

Comment:
These are *not* postcard serialized and myay even be safe to remove since we likely don't have any in the module store, but the safest is to leave them for now

Reply:
Yeah I kept these for the conversion. Might change the comment to be more clear.